### PR TITLE
Unwrap CmdError

### PR DESCRIPTION
**Pull request by joshmoore**
metrics.py was failing with the following error which
left the underlying reason unclear:

```
 systemd[1]: Starting prometheus-omero-py...
 python[18153]: Login succeeded
 python[18153]: Traceback (most recent call last):
 python[18153]: File "/opt/prometheus-omero-py/src/sessions/metrics.py", line 96, in
 python[18153]: metrics.update()
 python[18153]: File "<decorator-gen-1>", line 2, in update
 python[18153]: File "/opt/prometheus-omero-py/venv/lib/python2.7/site-packages/prome
 python[18153]: return func(*args, **kwargs)
 python[18153]: File "/opt/prometheus-omero-py/src/sessions/metrics.py", line 59, in
 python[18153]: cb = client.submit(omero.cmd.CurrentSessionsRequest(), 60, 500)
 python[18153]: File "/opt/omero/server/OMERO.server/lib/python/omero/clients.py", li
 python[18153]: closehandle=True)
 python[18153]: File "/opt/omero/server/OMERO.server/lib/python/omero/clients.py", li
 python[18153]: raise omero.CmdError(rsp)
 python[18153]: omero.CmdError
```

### CODE REVIEW

File: sessions/metrics.py [hunk 1]
```diff
@@ -70,8 +72,12 @@ def update(self):
                         print('%s: %d' % (username, n))
                     self.g_sessions.labels(username).set(n)
                     self.lastusers.add(username)
+            except omero.CmdError, ce:
```
Comment (manics) [inline sessions/metrics.py]: Can you use `as ce` instead for python 3 compatibility?

Comment (joshmoore) [reply sessions/metrics.py]: Force pushed.

**Review by manics — APPROVED** [review]

**Comment by manics** [comment]
Optional: if you want to make the journald log message slightly more friendly you can add https://github.com/manics/omero-prometheus-tools/commit/b56bca9bf47e21ed1e64866454206420dff8dde4

**Comment by joshmoore** [comment]
Merging this for a redeployment on prod50.

**Pull request merged**